specification/workloads/resource-manager/Microsoft.Workloads/monitors/preview/2024-02-01-preview/monitors.json

```diff
@@ -0,0 +1,2805 @@
+            "schema": {
+              "$ref": "#/definitions/Alert"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource 'Alert' update operation succeeded",
+            "schema": {
+              "$ref": "#/definitions/Alert"
+            }
+          },
+          "201": {
+            "description": "Resource 'Alert' create operation succeeded",
+            "schema": {
+              "$ref": "#/definitions/Alert"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string",
+                "description": "A link to the status monitor"
+              },
+              "Retry-After": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Creates an Alert": {
+            "$ref": "./examples/Alerts_Create.json"
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "operationId": "Alerts_Delete",
+        "tags": [
+          "Alerts"
+        ],
+        "description": "Deletes an alert for the specified subscription, resource group, SAP monitor name, and resource name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "monitorName",
+            "in": "path",
+            "description": "Name of the SAP monitor resource.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-_]+$"
+          },
+          {
+            "name": "alertName",
+            "in": "path",
+            "description": "Name of the SAP monitor alert resource.",
+            "required": true,
+            "type": "string",
+            "minLength": 1,
+            "pattern": "^[a-zA-Z0-9-_]+$"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Resource deletion accepted.",
+            "headers": {
+              "Location": {
```

Comment:
>Location [](http://example.com/codeflow?start=15&length=8)

Out of curiosity, why Alerts_Create 201 has Azure-AsyncOperation while Alerts_Delete 202 has Location header.